ui/src/app/tags/views/TagUpdate/TagUpdate.tsx

```diff
@@ -91,7 +93,19 @@ const TagUpdate = ({ id }: Props): JSX.Element => {
         dispatch(tagActions.cleanup());
         dispatch(tagActions.update(values));
       }}
-      onSuccess={onClose}
+      onSuccess={(values) => {
+        if (values.definition !== initialValues.definition) {
```

Comment:
I think this is the same case as the comment you made on my PR about initial values not necessarily matching what's on the tag, so maybe this should be `values.definition !== tag.definition`.

Reply:
That's hilarious hahahaha, clearly I don't take my own advice